packages/frontend-2/components/common/TitleDescription.vue

```diff
@@ -0,0 +1,15 @@
+<template>
+  <div class="flex flex-col gap-2 sm:pr-8">
```

Comment:
I think we can get rid of this right padding now that the edit icon is gone?

Reply:
I added this padding to increase the spacing between the team panel and the title/description.

I can see how this was unclear, so I have removed the padding and instead increased the gap in the parent item, by removing the responsive helper `sm:gap-4`